playground/cypress/index.html

```diff
@@ -48,8 +48,12 @@
         Throw error
     </button>
 
+    <button data-cy-exception-button onclick="posthog.captureException(new Error('wat even am I'), { extra_prop: 2 })">
```

Comment:
Test currently failing because the `posthog.captureException` method cannot be found. Does this file rely on the local instance of PostHog or remote?

Reply:
might need to update the snippet in the cypress html?